src/LibTeXPrintf.jl

```diff
@@ -10,22 +10,27 @@ using .libtexprintf
 include("capture.jl")
 
 
-export stexprintf, texprintf, texsymbols, texfonts, texsetfont, texgetfont
+export stexprintf, texprintf, texsymbols, texfonts, texsetfont!, texgetfont, texsetascii, texsetunicode
 
 function __init__()
-    # this is a workaround so `stdout` can work correctly and be captured by Suppressor
-    # libtexprintf.texprintf("")
+    texsetfont!("text")
     TEXSYMBOLS[] = ImmutableDict([
         if length(line) == 4
             String(line[2] * line[3]) => String(line[4])
         else
             String(line[2]) => String(line[3])
         end for line in split.(split(strip(capture_out() do
                     libtexprintf.texlistsymbols()
+                    # this is a workaround so `stdout` can work correctly and be captured by
+                    # Suppressor, this work by forcing a fflush on Libc using texerrors()
+                    libtexprintf.texerrors()
```

Comment:
This doesn't need to be done in `__init__`, it can be done at the top level — it doesn't need to run every time the package loads, just during precompilation, since the symbols don't change at load-time.

Reply:
i.e. 
```jl
const TEXSYMBOLS = Base.ImmutableDict(let
    texsetfont!("text")
    [ ... ]
end)
```

Reply:
Yup, originally I had it like, except the non-reproducible `stdout` capture error cause it only to work in `__init__`. 😅

Reply:
I just sent a patch to upstream to expose `texsymbols_str` which returns the list formated as `"command:symbol;"` in a string that needs to be freed with `texfree`.

Reply:
For now the error if tryied to do it in precompile-time still happens, so I'll update that later.